scripts/update_csv.py

```diff
@@ -48,42 +47,37 @@
 # spec version
 csv['spec']['version'] = newversion
 
-# updating site-controller container image
-csv['spec']['install']['spec']['deployments'][0]['spec']['template']['spec']['containers'][0]['image'] = sitecontrollersha
+# updating controller container image
+csv['spec']['install']['spec']['deployments'][0]['spec']['template']['spec']['containers'][0]['image'] = controllersha
 
 # updating images for environment variables
 for envvar in csv['spec']['install']['spec']['deployments'][0]['spec']['template']['spec']['containers'][0]['env']:
     if envvar['name'] == "QDROUTERD_IMAGE":
         envvar['value'] = routersha
-    elif envvar['name'] == "SKUPPER_SERVICE_CONTROLLER_IMAGE":
-        envvar['value'] = servicecontrollersha
-    elif envvar['name'] == "SKUPPER_SITE_CONTROLLER_IMAGE":
-        envvar['value'] = sitecontrollersha
-    elif envvar['name'] == "SKUPPER_CONFIG_SYNC_IMAGE":
-        envvar['value'] = configsyncsha
-    elif envvar['name'] == "SKUPPER_FLOW_COLLECTOR_IMAGE":
-        envvar['value'] = flowcollectorsha
+    elif envvar['name'] == "SKUPPER_CONTROLLER_IMAGE":
+        envvar['value'] = controllersha
+    elif envvar['name'] == "SKUPPER_KUBE_ADAPTOR_IMAGE":
+        envvar['value'] = kubeadaptorsha
+    elif envvar['name'] == "SKUPPER_NETWORK_OBSERVER_IMAGE":
```

Comment:
Maybe we will need to add a new var (on skupper repo) to the nginx image used by the observer.